package.json

```diff
@@ -13,6 +13,8 @@
     "test": "npm run lint --silent && npm run unit && npm run closure",
     "watch": "find . -name '*.js' -not -path '*/node_modules/*' -not -path '*/extension/*' | entr npm run unit",
     "unit": "mocha $(find ./test -name '*.js') --timeout 60000",
+    "coverage": "istanbul cover -x \"**/third_party/**\" _mocha -- $(find ./test -name '*.js') --timeout 60000",
```

Comment:
other people out there swear that `istanbul cover npm run unit` should work, but I couldn't ever get coverage information out of it. Hence the c/p